technical-reference/reference-technical-articles/multichain-governance.md

```diff
@@ -99,6 +100,17 @@ It's crucial to understand that the proposal ID for the remote execution on the
 * `TimelockV8`:
   * Owned by: `OmnichainGovernanceExecutor` contract. This ownership grants `TimelockV8` the authority to perform specific actions like queuing, canceling, and executing remote proposals.
 
+## Potential Failures and Retry Options
+* If a VIP fails to send the proposal from source chain (BNB Chain) to destination network, the message will be saved and can be retried with the `retryExecute` function. This mechanism allows for the redelivery of the message with potentially adjusted gas fees to ensure successful execution.
+
+* In case where the VIP fails to send the proposal from source chain (BNB Chain) to destination network due to non-retryable conditions and funds are stuck in the `OmnichainProposalSender` contract, funds can be withdrawn by invoking the `fallbackWithdraw` function. This ensures that funds are not permanently locked in the contract due to failed execution attempts.
+
+* If a proposal's queueing fails on the destination network due to insufficient gas, the message will be saved and can be retried with the `retryMessage` function.
+
+* If a proposal's queueing fails on the destination network due to reasons such as reaching caps or other constraints, the message will be saved and can be retried with the `retryMessage` function. This provides flexibility in addressing various failure scenarios and ensures that execution attempts are made until successful.
+
+* If a proposal's queueing fails on the destination network because the contract has been paused, the message will be saved and can be retried with the'`retryMessage`' function after the contract has been unpaused.
```

Comment:
```suggestion
* If a proposal's queueing fails on the destination network because the contract has been paused, the message will be saved and can be retried with the '`retryMessage`' function after the contract has been unpaused.
```

Reply:
https://github.com/VenusProtocol/venus-protocol-documentation/compare/41d73bdc17443d2b317e1bf7037cfb3c9f22c053..20dc883a4bfcd35af186e53ace6cdc6e8a649c05